src/components/select/__internal__/select-list/select-list.style.ts

```diff
@@ -72,7 +72,7 @@ const StyledSelectListTableHeader = styled.thead<StyledSelectListTableHeaderProp
     padding: var(--spacing200);
     background-color: white;
     text-align: left;
-    font-weight: 900;
+    font-weight: 500;
```

Comment:
**suggestion(non-blocking)**: similarly did we intend for this to the boldest?:
```suggestion
    font-weight: 700;
```

Reply:
Yes I meant it to be 500, the weight was too thick for what it was being used for.

Reply:
Ah brilliant, thanks for clarifying this @harpalsingh 👍🏼